Upgrade to latest neon driver

cramforce opened the pull request:
- Had to fix a kysely test due to brittle mocking
- Fixed playwright tests (were just broken)
- Fixed Vercel turbo build (was just broken)

cramforce committed:
Upgrade to latest neon driver

cramforce committed:
Attempt to fix turbo build

cramforce committed:
Remove playwright caching

cramforce committed:
Fix unit tests

cramforce committed:
Update playwright

vvo commented on turbo.json:
```diff
@@ -3,7 +3,7 @@
     "build": {
       "dependsOn": ["^build"],
-      "outputs": ["dist/**"]
+      "outputs": [".next/**", "!.next/cache/**", "dist/**"]
```
This change errors with:

```
ENOENT: no such file or directory, lstat '/vercel/path0/node_modules/.pnpm/next@13.4.3_@babel+core@7.21.0_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js'
```

It happens on the first deployment of every PR, we had a lot of issues with this. We have no idea how to fix it though (we asked around).

cc @cramforce

vvo committed:
chore(CI): use local playwright version for install

Otherwise we end up with messages like:

```
Error: browserType.launch: Executable doesn't exist at...
Looks like Playwright Test or Playwright was just installed or updated.
```

Because dlx never uses the local version and always use the remote.
Because when there's a playwright update, it will download new browser versions
while we've tried to use the local cache.

vvo committed:
add comment

vvo committed:
Create tricky-taxis-raise.md

vvo committed:
fix changeset
